lib/web_api.ex

```diff
@@ -0,0 +1,85 @@
+defmodule WebApi do
+  use Plug.Router
+  require Logger
+
+  plug(Plug.Parsers, parsers: [{:json, json_decoder: Jason}])
+  plug(:match)
+  plug(:enforce_api_key)
+  plug(:dispatch)
+
+  post "/tts" do
+    %{"text" => text, "lang" => lang} = conn.body_params
+
+    voice_id =
```

Comment:
Could this be something that can come from the request and default to these if absent? Might be thinking a bit too far in the future here, but If the Screens app were to eventually use this service, they use a different voice. Could give a bit more flexibility to apps using the service.

Reply:
Makes sense. I wrote it like this thinking we wanted to standardize on a particular voice, but if we want different ones for different apps then we'll need to offer that flexibility.